sdks/typescript/container/boot.go

```diff
@@ -169,13 +169,14 @@ func main() {
 		args = append(args, "--status_endpoint="+info.GetStatusEndpoint().GetUrl())
 	}
 
-  workerIds := append([]string{*workerId}, info.GetSiblingWorkerIds()...)
+  workerIds := append([]string{*id}, info.GetSiblingWorkerIds()...)
 	var wg sync.WaitGroup
 	wg.Add(len(workerIds))
 	for _, workerId := range workerIds {
 		go func(workerId string) {
-			log.Printf("Executing: python %v", strings.Join(args, " "))
-			log.Fatalf("User program exited: %v", execx.ExecuteEnv(map[string]string{"WORKER_ID": workerId}, "npx", args...))
+		  workerArgs := append(append([]string{}, args...), "--id=" + workerId)
```

Comment:
Looks like this file might need a `go fmt`

Reply:
Done.